src/Opaleye/Order.hs

```diff
@@ -72,13 +72,39 @@ descNullsLast = O.order HPQ.OrderOp { HPQ.orderDirection = HPQ.OpDesc
 {- |
 Limit the results of the given query to the given maximum number of
 items.
+
+/WARNING:/ If you're planning on using limit/offset together please use
+'offset' /before/ you use 'limit'. eg:
+
+@
+limit 10 $ offset 50 yourQuery
```

Comment:
should thee be `limitOffset 10 50` operator, to make this common use case more obvious?

Reply:
Maybe.  See https://github.com/tomjaguarpaw/haskell-opaleye/issues/341#issuecomment-341110622